src/main/java/hudson/remoting/InitializeJarCacheMain.java

```diff
@@ -34,7 +34,7 @@ public boolean accept(File dir, String name) {
     public static void main(String[] argv) throws Exception {
         if (argv.length != 2) {
             throw new IllegalArgumentException(
-                    "Usage: java -cp slave.jar hudson.remoting.InitializeJarCacheMain " +
+                    "Usage: java -cp agent.jar hudson.remoting.InitializeJarCacheMain " +
```

Comment:
It's compliant with https://github.com/jenkinsci/jenkins/pull/3004 from @ndeloof . LGTM though I would prefer to just use `remoting.jar` in the documentation

Reply:
Sorry, what do you mean by "documentation"? I can change this to remoting.jar if you prefer.

Reply:
Let's keep it as is for now. It's definitely a step forward, switching to `remoting.jar` is a separate story (which is probably not required at all)